lib/reverse_sentence.rb

```diff
@@ -1,6 +1,56 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n) overall.  I think Str[0...-1] is O(k) where k is based on avg word size and therefore constant
+# Space complexity: O(n)
```

Comment:
You're not doing this reverse in-place, instead you are creating a new array.  Could you do this in-place without creating an auxiliary array?

Reply:
omg i didn't follow directions, oops. will try to fix...

Reply:
FIXED, sorry Chris!!!